src/box/txn_limbo.c

```diff
@@ -40,6 +40,70 @@
 
 struct txn_limbo txn_limbo;
 
+static int
+synchro_request_write(const struct synchro_request *req);
+
+static void
+txn_limbo_read_confirm(struct txn_limbo *limbo, int64_t lsn);
+
+/**
+ * Write a confirmation entry to the WAL. After it's written all the
+ * transactions waiting for confirmation may be finished.
+ */
+static int
+txn_limbo_write_confirm(struct txn_limbo *limbo)
```

Comment:
Nit: it seems better to introduce `txn_limbo_write_synchro` and `txn_limbo_write_synchro_or_panic`. Since in the future not only confirm may become repeatable and we won't introduce the same function for each request.

Reply:
Done.